### DISCUSSION TITLE
About the replacement of lazer as the new stable

### DISCUSSION THREAD

**Discussion by Ladwick** [ppy/osu#24253] [Q&A] [answered]
Since the 'hard links' were introduced to lazer I have this question that I really can't answer myself.

Basically, since the hard links are a way to create a 'duplicate' to the maps in the stable folder without taking space in disk, just as if they were a 'shortcut' for the original song (or at least that's what I understand), what will happen if, for example, lazer became the next stable tomorrow and I wanted to completely delete the old stable? Should I have to re-download ALL the maps in lazer since deleting the old stable client and folders would delete also the duplicates for lazer? I'm really curious about how that will work, if someone could answer I would be pretty happy because for people that have a huge load of maps, doing that would be complete pain, unless they add back the old way of importing maps so all the maps just duplicate into lazer?

**Comment by peppy** [comment, marked as answer]
Hard links exist at both places. If you delete one it will not affect the other.

**Reply by Ladwick** [reply]
Thank you for your answer!
